app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <dimen name="audio_fragment_corner_radius">8dp</dimen>
+  <dimen name="margin_32dp">32dp</dimen>
```

Comment:
Why have this dimension if it just says 32dp? Ditto below for the 8dp one.

Dimensions should have semantic purpose, and be shared. If a dimension isn't shared, we should inline the value at the place it's used. Extra dimensions just adds bloat to the binary with no gain. If a dimension should be shared among multiple views, then we should add it to dimens.

Reply:
I created this because I was thinking even if in future we have to change the value we can reference it from there, but now after your comment I realised whatever I was thinking and even if change this margin_32dp will still be redundant. So removing this and using 32dp directly.